server/routes/upload.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+const multer = require('multer');
+const {uploadPhotos} = require('../controllers/upload');
+// const {check_max_photos} = require('../middleware/upload');
+
+// const {check_max_photos} = require('../middleware/upload')
```

Comment:
You can remove commented out code here

Reply:
Sorry, forgot to share it on Slack, I'll do that right now